litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts

```diff
@@ -31,12 +28,9 @@ export default [
     title: 'kube-proxy-chaos',
     chaosinfra: true,
     urlToIcon: 'https://hub.litmuschaos.io/api/icon/1.8.0/generic/generic.png',
-    chaosWkfCRDLink:
-      'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/kube-proxy-all/workflow.yaml',
-    chaosWkfCRDLink_Recur:
-      'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/kube-proxy-all/workflow_cron.yaml',
-    gitLink:
-      'https://github.com/litmuschaos/chaos-charts/blob/master/workflows/kube-proxy-all',
+    chaosWkfCRDLink: `https://raw.githubusercontent.com/litmuschaos/chaos-charts/${process.env.REACT_APP_HUB_BRANCH_NAME}/workflows/kube-proxy-all/workflow.yaml`,
```

Comment:
Not only related to this PR. But why are we retrieving manifest from the backend as well from the raw git? Can't we only retrieve from the backend?

Reply:
We are fetching the manifest url for pre-defined experiments using this way in frontend. This approach will be changed in the next release since we can get the data from my hub as well. cc: @ishangupta-ds @gdsoumya

Reply:
From the backend(myhub) we are getting the manifest from the charts directory but the predefined experiemnts are present in a different directory. Will be creating a separate API to get the pre-defined workflows from myhub and this file will be removed.